core/embed/rust/src/ui/layout_eckhart/ui_firmware.rs

```diff
@@ -1197,17 +1195,35 @@ impl FirmwareUI for UIEckhart {
 
     fn show_device_menu(
         failed_backup: bool,
+        pin_unset: bool,
```

Comment:
nit: can we remove `pin_unset: bool` and replace it with `!pin_code.unwrap_or_default()`?